### PR TITLE
Simplify apt bazel rules

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: package_managers/download_pkgs.bzl [hunk 1]
```diff
@@ -14,11 +14,26 @@
...
+# Fetch Index
+apt-get update -y
+# Make partial dir
+mkdir -p {cache}/{archive}/partial
+# Install command
+apt-get install --no-install-recommends -y -q -o Dir::Cache="{cache}" -o Dir::Cache::archives="{archive}" {packages} --download-only
+# Tar command to only include all the *.deb files and ignore other directories placed in the cache dir.
+tar -cpf {output}.tar --directory {cache}/{archive} `cd {cache}/{archive} && ls *.deb`""".format(
+    output=ctx.attr.name,
+    cache='/tmp/install',
```
Comment (dlorenc) [inline package_managers/download_pkgs.bzl]: nit: can you just hardcode the ones that we have hardcoded?

File: package_managers/install_pkgs.bzl [hunk 2]
```diff
@@ -16,18 +16,22 @@
 
 load("//package_managers:package_manager_provider.bzl", "package_manager_provider")
 
+def _generate_install_commands(tar):
+  return """
+tar -xvf {}
```
Comment (dlorenc) [inline package_managers/install_pkgs.bzl]: could you name this template parameter?

File: package_managers/download_pkgs.bzl (hunk 1 above)
Comment (sharifelgamal) [reply package_managers/download_pkgs.bzl]: Done.

File: package_managers/install_pkgs.bzl (hunk 2 above)
Comment (sharifelgamal) [reply package_managers/install_pkgs.bzl]: Done.

**Pull request merged**